tests/cloudformation/checks/resource/aws/test_IMDSv1Disabled.py

```diff
@@ -0,0 +1,27 @@
+import os
+import unittest
+
+from checkov.cloudformation.checks.resource.aws.IMDSv1Disabled import check
+from checkov.cloudformation.runner import Runner
+from checkov.runner_filter import RunnerFilter
+
+
+class TestIMDSv1Disabled(unittest.TestCase):
+
+    def test_summary(self):
+        runner = Runner()
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+
+        test_files_dir = current_dir + "/example_IMDSv1Disabled"
+        report = runner.run(root_folder=test_files_dir,runner_filter=RunnerFilter(checks=[check.id]))
+        summary = report.get_summary()
+
+        self.assertEqual(report.failed_checks[0].check_id, check.id)
+        self.assertEqual(summary['passed'], 2)
+        self.assertEqual(summary['failed'], 3)
```

Comment:
Please check the IDs match the expected pass / fail to avoid inversions. Example implementation:
https://github.com/bridgecrewio/checkov/blob/77c27f1d5e4f203ab9c49528fad04ab302bad5d3/tests/kubernetes/checks/test_ApiServerServiceAccountPlugin.py#L25

Reply:
Added this loop for pass and fails.